app/models/image-set-marker.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Object.extend({
+  imageSet: null,
+  lat: Ember.computed.alias('imageSet.latitude'),
```

Comment:
I think a better solution is like we had in gwatt settings, where we have intermediate variables selectedLatitude, selectedAge, etc. so we set them when the user presses save, and they also have the option to cancel.